### DISCUSSION TITLE
Standalone to Grid-Manager

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2443] [Unsupported Versions] [answered]
I originally installed my system as a Standalone, but have since wanted it to function as a Grid Manager also.  Do I need to modify something?  

I installed a Forward Node and pointed it to the Standalone server, but it didn't seem to work and the Node had errors during install.

Do I need to reinstall the primary server and define it as a Grid Manager?  I'd prefer not to have to reinstall.

**Comment by TOoSmOotH** [comment]
What were the errors? You can add nodes to standalone.

**Comment by innovate-support** [comment]
Error says 'Attempt to authenticate with the salt master failed with timeout error'

**Comment by innovate-support** [comment]
Manager status shows all OK.  SSH is enabled and functioning.  Is there some other firewall ports I need open?

**Comment by innovate-support** [comment]
I ran the so-setup-network on the Forward Node again and went through the setup again.  At the final 'Verifying setup' status bar shows 95% and then says 'Install had a problem. See /root/sosetup.log for details.  The end of the log says the same error of 'Attempt to authenticate with the salt master failed with timeout error'

**Comment by innovate-support** [comment]
I checked /opt/so/saltstack/local/salt/firewall/hostgroups.local.yaml on the manager and it shows the correct Manager and Forward Node ip addresses in the minion section.

I am also able to successfully SSH from Node to Manager without issues.

**Comment by innovate-support** [comment, marked as answer]
So I noticed the Manager had already updated to v2.3.21 and the Node installation I was using was v2.3.2

So I downloaded the 2.3.21 ISO and reinstalled the Node from scratch one more time.  This time it stalled at 95% Verifying Setup for quite a while, and then finished successfully.  It magically appeared in the 'Grid' section of the Manager web interface, and also stats showed up in Grafana.  I guess versions matter.
